<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setGameDlg</class>
 <widget class="QDialog" name="setGameDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>421</width>
     <height>181</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;设置游戏模式&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tabWidgetPage1">
    <property name="contextMenuPolicy">
     <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
    </property>
    <attribute name="title">
     <string>游戏模式选择</string>
    </attribute>
    <widget class="QRadioButton" name="modeChoseButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>常规模式</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="modeChoseButton_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>平衡模式</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>63</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>难度选择</string>
     </property>
    </widget>
    <widget class="QComboBox" name="chooseDimensionBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>8×8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10×10</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="chooseStoneNumBox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>80</y>
       <width>80</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>63</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>难度选择</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>游戏声音选择</string>
    </attribute>
    <widget class="QRadioButton" name="DefaltMusicButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>110</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>使用默认音乐</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="tailorMusicButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>使用自定义音乐</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="musicFileChooseEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>201</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="chooseMusicFileButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>选择音乐文件</string>
     </property>
    </widget>
    <widget class="QSlider" name="volumeSlider">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>设置音量</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>背景图片设置</string>
    </attribute>
    <widget class="QComboBox" name="comboBox_gemtype">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>宝石1</string>
      </property>
      <property name="icon">
       <iconset resource="resouce.qrc">
        <normaloff>:/pics/resource/gemOrange.png</normaloff>:/pics/resource/gemOrange.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>宝石2</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../1FinalProject/作业/大二上实训/cnamedlg等118个文件/gem1.png</normaloff>../1FinalProject/作业/大二上实训/cnamedlg等118个文件/gem1.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>宝石3</string>
      </property>
      <property name="icon">
       <iconset resource="resouce.qrc">
        <normaloff>:/pics/resource/mine5.png</normaloff>:/pics/resource/mine5.png</iconset>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_background">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>背景1</string>
      </property>
      <property name="icon">
       <iconset resource="resouce.qrc">
        <normaloff>:/pics/resource/back1.png</normaloff>:/pics/resource/back1.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>背景2</string>
      </property>
      <property name="icon">
       <iconset resource="resouce.qrc">
        <normaloff>:/pics/resource/back2.png</normaloff>:/pics/resource/back2.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>背景3</string>
      </property>
      <property name="icon">
       <iconset resource="resouce.qrc">
        <normaloff>:/pics/resource/back3.png</normaloff>:/pics/resource/back3.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>背景4</string>
      </property>
      <property name="icon">
       <iconset resource="resouce.qrc">
        <normaloff>:/pics/resource/back4.png</normaloff>:/pics/resource/back4.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>背景5</string>
      </property>
      <property name="icon">
       <iconset resource="resouce.qrc">
        <normaloff>:/pics/resource/back5.png</normaloff>:/pics/resource/back5.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>背景6</string>
      </property>
      <property name="icon">
       <iconset resource="resouce.qrc">
        <normaloff>:/pics/resource/back6.png</normaloff>:/pics/resource/back6.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>背景7</string>
      </property>
      <property name="icon">
       <iconset resource="resouce.qrc">
        <normaloff>:/pics/resource/back7.png</normaloff>:/pics/resource/back7.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>背景8</string>
      </property>
      <property name="icon">
       <iconset resource="resouce.qrc">
        <normaloff>:/pics/resource/back8.png</normaloff>:/pics/resource/back8.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>背景9</string>
      </property>
      <property name="icon">
       <iconset resource="resouce.qrc">
        <normaloff>:/pics/resource/back9.jpg</normaloff>:/pics/resource/back9.jpg</iconset>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>260</y>
     <width>86</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>开始游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>86</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resouce.qrc"/>
 </resources>
 <connections/>
</ui>
